src/modules/billing/services/cm-refresh-service.js

```diff
@@ -106,8 +106,7 @@ const mapTransaction = (transactionMap, cmTransaction) => {
     : new Transaction();
 
   return transaction.fromHash({
-    ...transactionMapper.cmToPojo(cmTransaction),
-    isTwoPartTariffSupplementary: false
+    ...transactionMapper.cmToPojo(cmTransaction)
```

Comment:
No need to itinerate through object since we're only returning a single thing.

Consider:
```
return transaction.fromHash(transactionMapper.cmToPojo(cmTransaction));
```

Reply:
good catch thanks